lib/wallet/test/unit/Cardano/Wallet/DB/Store/Delegations/Migrations/V5Spec.hs

```diff
@@ -6,6 +6,9 @@ module Cardano.Wallet.DB.Store.Delegations.Migrations.V5Spec where
 
 import Prelude
 
+import Cardano.Api
+    ( SlotNo
+    )
```

Comment:
```suggestion
import Cardano.Slotting.Slot
    ( SlotNo
    )
```

I didn't catch it on the first review — could you take the `SlotNo` from here instead of the `cardano-api`? The `cardano-slotting` number is closer to the ledger and our `read` side.

Reply:
@HeinrichApfelmus what about enabling in codebase this type from cardano-slotting in `Cardano.Wallet.Primitive.Slot` in primitive lib?

Reply:
IMHO it should be probably here in this module -> https://github.com/cardano-foundation/cardano-wallet/blob/master/lib/primitive/lib/Cardano/Wallet/Primitive/Slotting.hs

Reply:
@paolino ^^^